### DISCUSSION TITLE
Couldnt get consistent frames style in text2video

### DISCUSSION THREAD

**Discussion by click2balaji** [volotat/SD-CN-Animation#63] [Q&A] [answered]
I tried to create a video from text2image, the output seems realistic at first and it transforms into abstract illustration style.

The are the prompts used to create the videos:
A simple one as "**Will Smith eating pizza**" and as detailed as mentioned in the example "**RAW photo, closeup view of a realistic flower, 8k uhd, dslr, soft lighting, high quality, film grain, Fujifilm XT3**".

Attached my settings screenshot and the video results, help me to produce full video in a specific style, dont know what am i missing.
https://github.com/volotat/SD-CN-Animation/assets/8522456/a26308a0-ed51-4905-b63c-72c254ba4f2a
https://github.com/volotat/SD-CN-Animation/assets/8522456/695296af-408f-4cd6-8afc-930bee70246c
![t2v1](https://github.com/volotat/SD-CN-Animation/assets/8522456/75f946da-49db-47b1-a138-9e9adc387461)
![t2v2](https://github.com/volotat/SD-CN-Animation/assets/8522456/cf49fe22-e166-423a-95ab-be1a1a70bd95)

**Comment by volotat** [comment, marked as answer]
Try to use this model https://civitai.com/models/4201/realistic-vision-v20 instead of v1-5-inpainting. For me this model works in a much more stable way.

**Reply by click2balaji** [reply]
It worked, thank you!

Check out this updated result.
https://github.com/volotat/SD-CN-Animation/assets/8522456/de740d1d-2c82-41f9-b4e8-e8113886741d

**Reply by volotat** [reply]
Doesn't loads for me unfortunately. :(
